.travis.yml

```diff
@@ -49,13 +49,13 @@ env:
     - "TEST_SUITE=karma"
     - "TEST_SUITE=protractor"
     # mysql2
-    - "TEST_SUITE=cucumber DB=mysql2"
-    - "TEST_SUITE=test     DB=mysql2"
-    - "TEST_SUITE=spec     DB=mysql2"
+    - "TEST_SUITE=cucumber    DB=mysql2"
+    - "TEST_SUITE=spec:core   DB=mysql2"
```

Comment:
Worth noting that although `TEST_SUITE=spec` would work fine here (albeit slowly, a run taking ~25 mins), I split the suites into two again in a5c3dace66ea90c3158002e8a2353f4da3b50f18 to take advantage of [Travis's parallelisation](http://docs.travis-ci.com/user/speeding-up-the-build/). This is something we might also want to consider for our internal CI.

It'd actually make more sense IMO to split up unit and feature specs, especially since the latter are more prone to flickering:

```yaml
  - "TEST_SUITE=spec:models  DB=postgres"
  - "TEST_SUITE=spec:features DB=postgres"
```

However, this would warrant further discussion and is beyond the scope of this already huge PR.

/ @jonasheinrich @kgalli @NobodysNightmare @floriank